docs/plugins/Latex.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Latex"
+tags:
+  - plugin/transformer
+---
```

Comment:
these should all have prose descriptions about what they do (or link back to the feature page)

Reply:
Since the feature pages link here already, I added the sentence "See the backlinks for functionalities where this plugin is used." to all plugin pages. If you prefer, I can make it more explicit and include a direct link to all features that use it instead.

Reply:
Added a commit where I added explicit links insteas. Let me know if it needs more adjusting.